packages/http-signature-utils/README.md

```diff
@@ -19,6 +19,18 @@ npm install @interledger/http-signature-utils
 
 ## Usage
 
+Load a private Ed25519 key:
+
+```ts
+const key = parseKey('/PATH/TO/private-key.pem')
+```
+
+Generate a private Ed25519 key:
+
+```ts
+const key = provisionKey('/PATH_TO_SAVE_KEY_IN')
+```
+
```

Comment:
I followed the previous naming, but I wouldn't mind using `loadKey` and `generateKey` as well

Reply:
I don't think you necessarily need to change it but `loadKey` and `generateKey` are clearer to me.

Reply:
Yep, with you. Made this a major lib bump and updated the naming